sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -154,7 +154,7 @@ void _ottf_main(void) {
                                       .parity_enable = kDifToggleDisabled,
                                       .parity = kDifUartParityEven,
                                   },
-                                  kDifToggleEnabled));
+                                  kDifToggleEnabled, kDifToggleEnabled));
```

Comment:
This might have a slight readability issue akin to passing boolean values. To totally disambiguate, we could make it part of the config struct (so the named fields would do the trick), or we could add a comment showing the name of the function parameter.

I don't feel strongly about this, though.

Reply:
fixed.

Reply:
fixed